pages/events.js

```diff
@@ -72,6 +73,7 @@ export default function Events({ events }) {
       </Head>
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Community events</h1>
+        <Link href={'/docs/how-to-guides/events'} className="mx-auto md:mx-0 mb-4 hover:underline text-sky-600 md:text-lg">Learn how to add events</Link>
```

Comment:
you shouldn't need the curly braces here. I would also use double "" instead of single to align more with the style seen in links throughout the project.

Reply:
alright, i will remove this { }

Reply:
you can review now